docs/src/dev/developer/development-environment.asciidoc

```diff
@@ -497,6 +503,43 @@ docker/gremlin-server.sh 3.4.2
 To be a bit more clear, the version can not be a Docker tag like "latest" because there is no such TinkerPop artifact
 that has been published with that version number.
 
+[[docker-testing]]
+== Testing with Docker
+
+Some GLV allow you to run tests without setting up Java and without building of other parts of the server. Currently
+supported gremlin-go, gremlin-javascript and gremlin-dotnet.
```

Comment:
We can add the python and console stuff since we'll be PRing this with it

Reply:
added